src/microbe_stage/PlacedOrganelle.cs

```diff
@@ -607,6 +595,11 @@ private void SetupOrganelleGraphics()
         // Position the intermediate node relative to origin of cell
         var transform = new Transform(Quat.Identity,
             Hex.AxialToCartesian(Position) + Definition.CalculateModelOffset());
+        if (organelleMaterial != null)
+        {
+            organelleMaterial.RenderPriority = Hex.GetRenderPriority(Position);
+        }
+
```

Comment:
I'd put this after line 585, instead of here in *between* the transform setting.